site/content/docs/include/common-svc-fields.en.md

```diff
@@ -61,7 +61,7 @@ count:
 ```
 
 <span class="parent-field">count.</span><a id="count-range" href="#count-range" class="field">`range`</a> <span class="type">String or Map</span>  
-You can specify a minimum and maximum bound for the number of tasks your service should maintain.
+You can specify a minimum and maximum bound for the number of tasks your service should maintain, based on the values you specify for the metrics.
```

Comment:
Is 'metrics' referring to `cpu_percentage`, `memory_percentag`, `requests` and `response time`?

I feel like if we want to reword this, we might want to also mention that those metrics are not required. https://docs.aws.amazon.com/autoscaling/ec2/userguide/scaling_plan.html

I'm not sure what's the best way for us to explain how `range`, `cpu_percentage`, `memory_percentage`, etc. will interact.